package.json

```diff
@@ -179,6 +179,7 @@
     "raven-js": "^3.24.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-dropdown-select": "^4.3.0",
```

Comment:
This is necessary? i mean why we are adding a new library for a dropdown, we don't have any?

Reply:
It's a component that we can use for multiple purposes, its very used and have lots of functionalities. I think we should use it.
https://www.npmjs.com/package/react-dropdown-select

Reply:
We already have dropdown menus, at least on the contract interaction page (for selecting the contract method). So i will not add a new library for that. 

Also, the design specifies a searchbar nor a dropdown item